cri.md

```diff
@@ -87,8 +88,8 @@ repoDigests = []string{from.PreviousName + "@" + string(from.Digest)}
 This allows tools like `crictl` to output the image name by adding a `<none>`
 placeholder for the tag:
 
-```
-> sudo crictl images --digests
+```shell
+$ sudo crictl images --digests
```

Comment:
this seems like the only $ before a shell command. I don't mind either way but ideally they'd be consistent

Reply:
I noticed myself it was hard to stay consistent when going over all the files. I don't have a strong preference myself.
The reason why I included a `$` there is because the code block is mixed with command output and makes it easier to differentiate the command.

Reply:
I removed the `$` and split the code block up. I didn't even notice it was done like that right below.